NetKAN/FilterExtensions.netkan

```diff
@@ -18,6 +18,11 @@
         {
             "file"       : "GameData/000_FilterExtensions",
             "install_to" : "GameData"
+        },
+        {
+            "file"       : "GameData/000_FilterExtensions Configs"
+            "install_to" : "GameData"
```

Comment:
Oh no! We're missing commas here, and on the line above, which means our JSON isn't parsing.